<ui version="4.0" >
 <class>OpsAdvanced</class>
 <widget class="QWidget" name="OpsAdvanced" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>582</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>8</number>
   </property>
   <property name="topMargin" >
    <number>8</number>
   </property>
   <property name="rightMargin" >
    <number>8</number>
   </property>
   <property name="bottomMargin" >
    <number>8</number>
   </property>
   <item>
    <widget class="QCheckBox" name="kcfg_UseRefraction" >
     <property name="toolTip" >
      <string>Correct coordinates of objects for the effects of the atmosphere</string>
     </property>
     <property name="whatsThis" >
      <string>The atmosphere bends light passing through it, like a lens.  If this item is checked, this "atmospheric refraction" will be simulated in the sky map.  Note that this correction is only applied when using the Horizontal coordinate system.</string>
     </property>
     <property name="text" >
      <string>Correct for atmospheric refraction</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UseRelativistic" >
     <property name="toolTip" >
      <string>Correct for the effect of sun's gravity on star positions, as predicted by General Relativity, and verified by Eddington's experiment.</string>
     </property>
     <property name="text" >
      <string>General Relativity effects near the sun</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UseAnimatedSlewing" >
     <property name="toolTip" >
      <string>Show slewing motion when focus changes?</string>
     </property>
     <property name="whatsThis" >
      <string>If checked, changing the focus position will result in a visible animated "slew" to the new position.  Otherwise, the display will center on the new position instantaneously.</string>
     </property>
     <property name="text" >
      <string>Use animated slewing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UseAutoLabel" >
     <property name="toolTip" >
      <string>Show name label of centered object?</string>
     </property>
     <property name="whatsThis" >
      <string>If checked, a name label will be temporarily attached to an object while it is centered in the display.  You can attach a more persistent label to any object using the right-click popup menu.</string>
     </property>
     <property name="text" >
      <string>Attach label to centered object</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UseHoverLabel" >
     <property name="toolTip" >
      <string>Show name label of centered object?</string>
     </property>
     <property name="whatsThis" >
      <string>If checked, a name label will be temporarily attached to an object while it is centered in the display.  You can attach a more persistent label to any object using the right-click popup menu.</string>
     </property>
     <property name="text" >
      <string>Attach temporary label when hovering mouse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UseAntialias" >
     <property name="toolTip" >
      <string>Select this for smoother (but slower) graphics</string>
     </property>
     <property name="text" >
      <string>Use antialiased drawing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ObsListOptions" >
     <property name="title" >
      <string>Observing List Labels</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>8</number>
      </property>
      <property name="topMargin" >
       <number>8</number>
      </property>
      <property name="rightMargin" >
       <number>8</number>
      </property>
      <property name="bottomMargin" >
       <number>8</number>
      </property>
      <item>
       <widget class="QRadioButton" name="kcfg_ObsListSymbol" >
        <property name="text" >
         <string>Symbol</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kcfg_ObsListText" >
        <property name="text" >
         <string>Text</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
     <widget class="QCheckBox" name="kcfg_ObsListSaveImage" >
     <property name="toolTip" >
      <string>Save the downloaded DSS/SDSS images?</string>
     </property>
     <property name="whatsThis" >
      <string>Do you want the downloaded DSS/SDSS images to be saved when closing the observing list?</string>
     </property>
     <property name="text" >
      <string>Save downloaded images on exit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_HideOnSlew" >
     <property name="toolTip" >
      <string>Do not draw all objects while the map is moving?</string>
     </property>
     <property name="whatsThis" >
      <string>When the map is in motion, smooth animation is compromised if the program has too many objects to draw on the map; check this item to temporarily hide some of the objects while the display is in motion.</string>
     </property>
     <property name="text" >
      <string>Hide objects while moving</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabelHideTimeStep" >
       <property name="text" >
        <string>Also hide if time step larger than:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TimeStepBox" name="SlewTimeScale" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>111</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="HideBox" >
     <property name="title" >
      <string>Configure Hidden Objects</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>8</number>
      </property>
      <property name="topMargin" >
       <number>8</number>
      </property>
      <property name="rightMargin" >
       <number>8</number>
      </property>
      <property name="bottomMargin" >
       <number>8</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="kcfg_HideStars" >
          <property name="toolTip" >
           <string>Hide faint stars while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then fainter stars will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Stars fainter than</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MagnitudeSpinBox" name="kcfg_MagLimitHideStar" >
          <property name="minimumSize" >
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="whatsThis" >
           <string>Stars fainter than this will not be drawn while the map is moving.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabelMag1" >
          <property name="text" >
           <string>mag</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>240</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item rowspan="2" row="4" column="0" >
         <widget class="QCheckBox" name="kcfg_HideIC" >
          <property name="toolTip" >
           <string>Hide IC objects while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then objects in the IC catalog will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>IC objects</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QCheckBox" name="kcfg_HideMilkyWay" >
          <property name="toolTip" >
           <string>Hide the Milky Way contour while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then the Milky Way contour will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Milky Way</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QCheckBox" name="kcfg_HidePlanets" >
          <property name="toolTip" >
           <string>Hide solar system bodies while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then all solar system bodies will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Solar system</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QCheckBox" name="kcfg_HideCBounds" >
          <property name="text" >
           <string>Constellation boundaries</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="kcfg_HideMessier" >
          <property name="toolTip" >
           <string>Hide Messier objects when moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then objects in the Messier catalog will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Messier objects</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QCheckBox" name="kcfg_HideGrids" >
          <property name="toolTip" >
           <string>Hide coordinate grids while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then the coordinate grids will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Coordinate grids</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="2" column="0" >
         <widget class="QCheckBox" name="kcfg_HideNGC" >
          <property name="toolTip" >
           <string>Hide NGC objects while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then objects in the NGC catalog will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>NGC objects</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="1" >
         <widget class="QCheckBox" name="kcfg_HideCLines" >
          <property name="toolTip" >
           <string>Hide constellation lines while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then constellation lines will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Constellation lines</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="1" column="1" >
         <widget class="QCheckBox" name="kcfg_HideCNames" >
          <property name="toolTip" >
           <string>Hide constellation names while moving?</string>
          </property>
          <property name="whatsThis" >
           <string>If checked, then constellation names will be hidden when the map is in motion.</string>
          </property>
          <property name="text" >
           <string>Constellation names</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QCheckBox" name="kcfg_HideLabels" >
          <property name="text" >
           <string>Object labels</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>468</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnitudeSpinBox</class>
   <extends>KDoubleNumInput</extends>
   <header>widgets/magnitudespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>TimeStepBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/timestepbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_UseRefraction</tabstop>
  <tabstop>kcfg_UseAnimatedSlewing</tabstop>
  <tabstop>kcfg_UseAutoLabel</tabstop>
  <tabstop>kcfg_HideOnSlew</tabstop>
  <tabstop>kcfg_HideStars</tabstop>
  <tabstop>kcfg_MagLimitHideStar</tabstop>
  <tabstop>kcfg_HidePlanets</tabstop>
  <tabstop>kcfg_HideMessier</tabstop>
  <tabstop>kcfg_HideNGC</tabstop>
  <tabstop>kcfg_HideIC</tabstop>
  <tabstop>kcfg_HideMilkyWay</tabstop>
  <tabstop>kcfg_HideCNames</tabstop>
  <tabstop>kcfg_HideCLines</tabstop>
  <tabstop>kcfg_HideGrids</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
